docs/development/submitting-changes.rst

```diff
@@ -0,0 +1,84 @@
+Submitting Changes
+==================
+
+This page details how to make and submit changes to the `j5` codebase.
+
+Repositories
+------------
+
+The main repository for `j5` is available on GitHub_, and Pull Requests should be submitted there.
+
+There is an additional repository available on GitLab_, although this is a mirror of the GitHub repo, and changes
+should not be submitted there.
+
+.. _GitHub: https://github.com/j5api/j5
+.. _GitLab: https://gitlab.com/j5api/j5
+
+Publishing Changes
+------------------
+
+The first step to submit changes is to publish them so that other contributors can review and give feedback.
+
+The mechanism to publish your changes depends on whether you have write access to the repository. Write access
+is granted by the Steering Council to recurrent, or trusted contributors.
+
+* Contributors with write access should publish a branch on the repository.
```

Comment:
Would it be better to have everyone follow the same (less privileged) workflow? This could help us surface issues in things like CI that may not appear until someone outside the organisation try to contribute which would impeded us merging in there changes without some potentially unnecessary back and forth.

Reply:
I'm uncertain on this one. I was trying to document the current process that we vaguely follow.

I can see the merits to this definitely.

Reply:
So, I've been thinking about this a bit and have decided that I agree with you.

However, I am not confident that the less-privileged workflow actually works, so we should ensure it does before changing to it entirely.

Reply:
I have broken this out into #571, so that we can get these guidelines up explaining our current workflow. I'll get to checking the CI works for that later today :)